backend/internal/controller/server/msg_handler_test.go

```diff
@@ -0,0 +1,103 @@
+package server
+
+import (
+	"bytes"
+	"encoding/json"
+	"testing"
+
+	"github.com/HackYourCareer/SmartKickers/internal/model"
+	"github.com/stretchr/testify/assert"
+)
+
+type dispatcherMsg struct {
+	MsgType string  `json:"type,omitempty"`
+	TableID string  `json:"id,omitempty"`
+	X       float64 `json:"x,omitempty"`
+	Y       float64 `json:"y,omitempty"`
+	Goal    int     `json:"goal,omitempty"`
+}
+
+type initialResponse struct {
+	GameID string `json:"start,omitempty"`
+}
+
+func TestCreateResponse(t *testing.T) {
+	type args struct {
+		name          string
+		msgIn         dispatcherMsg
+		returnsNil    bool
+		expectedError string
+	}
+	tests := []args{
+		{
+			name: "initial message",
```

Comment:
What is this test actually testing, receiving, responding?

I can suggest something like "on initial message response with GameId"